vnext/Microsoft.ReactNative/XamlUIService.idl

```diff
@@ -23,18 +26,28 @@ namespace Microsoft.ReactNative {
     void DispatchEvent(XAML_NAMESPACE.FrameworkElement view, String eventName, JSValueArgWriter eventDataArgWriter);
 
     // This needs to be manually provided to the ReactInstanceSettings when using XamlIslands
-    DOC_STRING("Set XamlRoot for app. This must be manually provided to the ReactInstanceSettings when using XamlIslands.")
+    DOC_STRING(
+      "Set XamlRoot for app. This must be manually provided to the ReactInstanceSettings when using XamlIslands.")
```

Comment:
```suggestion
      "Sets the @Windows.UI.Xaml.XamlRoot element for the app. This must be manually provided to the @ReactInstanceSettings object when using XAML Islands so that certain APIs work correctly.\n"
      "For more information, see [Host WinRT XAML Controls in desktop apps (XAML Islands)](https://docs.microsoft.com/windows/apps/desktop/modernize/xaml-islands).")
 ```